frontend/src/model/Wizard.ts

```diff
@@ -1,10 +1,14 @@
 export type Wizard = {
-    id:string,
-    name:string,
-    gender:Gender,
-    attribute1:string,
-    attribute2:string,
-    attribute3:string
+    id: string,
+    name: string,
+    gender: Gender,
+    attribute1: Attribute1,
```

Comment:
Wha's the purpose of this Attribute Group?
You can't tell what's their overarching purpose by just using numbers